src/ontology/hp-edit.owl

```diff
@@ -177109,7 +177109,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0033376> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0033377> (Increased airway neuroendocrine cells)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0003-0113-912X>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:20884725") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:26474448") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:30521210") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:32558323") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:33306420") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033377> "Presence of increased numbers of bombesin-immuno-positive neuroendocrine cells (NECs) within distal airways. There are no formal criteria for an increase (which is also dependend on the sensitivity of the staining method), findings of neuroendocrine cells in at least 70% of bronchioles by lung biopsy and at least 10% NECs in an individual airway are consistent with the diagnosis of Neuroendocrine cell hyperplasia of infancy in the appropriate clinical setting. Increases are also seen in other clicnial settings. Neuroendocrine neoplasms of the lung encompass NE tumors (NETs), which split into typical and atypical carcinoids, and NE carcinomas (NECs).")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0003-0113-912X>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:20884725") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:26474448") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:30521210") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:32558323") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:33306420") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033377> "Presence of increased numbers of bombesin-immuno-positive neuroendocrine cells (NECs) within distal airways. There are no formal criteria for an increase (which is also dependent on the sensitivity of the staining method), findings of neuroendocrine cells in at least 70% of bronchioles by lung biopsy and at least 10% NECs in an individual airway are consistent with the diagnosis of Neuroendocrine cell hyperplasia of infancy in the appropriate clinical setting. Increases are also seen in other clicnial settings. Neuroendocrine neoplasms of the lung encompass NE tumors (NETs), which split into typical and atypical carcinoids, and NE carcinomas (NECs).")
```

Comment:
one change was correct, but missed 'clicnial'

Reply:
good catch, I can propose this fix upstream to typos-cli :+1:. fixed in this pr too now